ui/src/contexts/issuer-state.tsx

```diff
@@ -0,0 +1,79 @@
+import { Space, Typography, message } from "antd";
+import {
+  PropsWithChildren,
+  createContext,
+  useCallback,
+  useContext,
+  useEffect,
+  useMemo,
+  useState,
+} from "react";
+
+import { APIError } from "src/adapters/api";
+import { getStatus } from "src/adapters/api/issuer-state";
+import { useEnvContext } from "src/contexts/env";
+import { AsyncTask } from "src/utils/async";
+import { isAbortedError, makeRequestAbortable } from "src/utils/browser";
+
+interface State {
+  notifyChange: () => Promise<void>;
+  refreshStatus: () => Promise<void>;
+  status: AsyncTask<boolean, APIError>;
+}
+
+const StateContext = createContext<State>({
+  notifyChange: () => Promise.reject("The state context is not yet ready"),
+  refreshStatus: () => Promise.reject("The state context is not yet ready"),
+  status: { status: "pending" },
+});
+
+export function StateProvider(props: PropsWithChildren) {
+  const env = useEnvContext();
+  const [status, setStatus] = useState<AsyncTask<boolean, APIError>>({ status: "pending" });
+
+  const refreshStatus = useCallback(
+    async (signal?: AbortSignal) => {
+      const response = await getStatus({ env, signal });
+
+      if (response.isSuccessful) {
+        setStatus({ data: response.data, status: "successful" });
+      } else {
+        if (!isAbortedError(response.error)) {
+          void message.error(response.error.message);
+        }
+      }
+    },
+    [env]
+  );
+
+  const notifyChange = useCallback(() => {
+    void message.success({
```

Comment:
I strongly believe this should be `message.info`. Let's [clarify](https://www.figma.com/file/BE3tUbvtSB2EqmJsybpeFC?node-id=5808:184164#417295665) this with Liudas.

Reply:
Hi @Figmintojas could you quickly weigh in here? Basically, instead of this:

![Screenshot from 2023-04-20 12-19-48](https://user-images.githubusercontent.com/2691584/233338691-37c47227-5d49-408a-9307-b7e0bfb9bd30.png)

I am advocating to [keep messaging consistent](https://www.figma.com/file/BE3tUbvtSB2EqmJsybpeFC/Polygon-ID-Platform?node-id=5806-174811&t=92KxN4W0RYdvJLdK-0) by always accompany it with the operational feedback and changing the icon to represent the message type. Like this:

![Screenshot from 2023-04-20 12-21-29](https://user-images.githubusercontent.com/2691584/233338732-9ed3537e-12d6-4843-aede-c0a61e20fb33.png)

Reply:
I think we can go ahead with this. It's pretty obvious you're right @cneuro. The first msg is info, the second is a success op confirmation.